elections.md

```diff
@@ -101,4 +87,15 @@ treated as “brand free” activities.
 ## Emeritus Term
 
 Members of the steering committee will graduate to becoming Emeritus members of
-the steering committee.
\ No newline at end of file
+the steering committee.
+
+## Bootstrap recusal
```

Comment:
Do we want to make this language permanent?  That no sitting member of the SC should electioneer, in any way, for a running candidate (except themselves, should their seat be up for election)?

Reply:
That sounds way better, I'll amend